core/integration/module_call_test.go

```diff
@@ -1175,6 +1175,62 @@ func (ModuleSuite) TestCallByName(ctx context.Context, t *testctx.T) {
 		require.Equal(t, "hi from mod-b", strings.TrimSpace(out))
 	})
 
+	t.Run("local with absolute paths", func(ctx context.Context, t *testctx.T) {
+		c := connect(ctx, t)
+
+		ctr := c.Container().From(golangImage).
+			WithMountedFile(testCLIBinPath, daggerCliFile(t, c)).
+			WithWorkdir("/work/mod-a").
+			With(daggerExec("init", "--source=.", "--name=mod-a", "--sdk=go")).
+			WithNewFile("/work/mod-a/main.go", dagger.ContainerWithNewFileOpts{
+				Contents: `package main
+
+			import "context"
+
+			type ModA struct {}
+
+			func (m *ModA) Fn(ctx context.Context) string {
+				return "hi from mod-a"
+			}
+			`,
+			}).
+			WithWorkdir("/work/mod-b").
+			With(daggerExec("init", "--source=.", "--name=mod-b", "--sdk=go")).
+			WithNewFile("/work/mod-b/main.go", dagger.ContainerWithNewFileOpts{
+				Contents: `package main
+
+			import "context"
+
+			type ModB struct {}
+
+			func (m *ModB) Fn(ctx context.Context) string {
+				return "hi from mod-b"
+			}
+			`,
+			}).
+			WithWorkdir("/work").
+			With(daggerExec("init")).
+			With(daggerExec("install", "--name", "foo", "/work/mod-a")).
+			With(daggerExec("install", "--name", "bar", "/work/mod-b"))
```

Comment:
Couple notes:
- we shouldn't have absolute paths in the `dagger.json`, the test should check for this - if absolute paths are allowed here, then the module isn't really reusable by anyone else on a different fs layout
- we still shouldn't load modules from above the root - I *think* that's what's happening here? we'd want an equivalent of the [dep points out of root](https://github.com/dagger/dagger/blob/a9155f4b4e68c264e673bbb2a0271701d76c5eb3/core/integration/module_config_test.go#L116-L161) test

Reply:
> we shouldn't have absolute paths in the dagger.json, the test should check for this - if absolute paths are allowed here, then the module isn't really reusable by anyone else on a different fs layout

Thanks, I updated the test to include that case 🙏 

> we still shouldn't load modules from above the root - I think that's what's happening here? we'd want an equivalent of the [dep points out of root](https://github.com/dagger/dagger/blob/a9155f4b4e68c264e673bbb2a0271701d76c5eb3/core/integration/module_config_test.go#L116-L161) test
I think the test is actually correct, as dep is a relative child inside the /work top level module

I however update the deps points out of root to handle those cases. The error message are not the best, but, we error correctly on the cases I anticipated